src/container/restly.jsx

```diff
@@ -0,0 +1,65 @@
+import React, { Component } from 'react';
+import HistoryList from '../components/app/rest-ly/History/HistoryList';
+import Controls from '../components/Controls/Controls';
+import Display from '../components/Display/Display';
+import { apiFetch } from '../services/apiFetch';
+
+export default class Restly extends Component {
+  state = {
+    url: '',
+    body: '',
+    method: '',
+    histories: [],
+    display: { 'Restly': 'Ready for request' }
+  }
+
+  handleChange = ({ target }) => {
+    this.setState({ [target.name]: target.value });
+  }
+
+  handleSubmit = async(event) => {
+    event.preventDefault();
+
+    await this.handleFetch();
+
+    return this.setState({
+      histories: [...this.state.histories, {
+        url: this.state.url,
+        body: this.state.body,
+        method: this.state.method,
+        key: `${this.state.url}+${this.state.method}`
+      }]
+    });
+  }
+
+  handleFetch = () => {
```

Comment:
This works but those pieces of state should be parameters for this function and you pass them in when you call it above